apps/portals/standards/src/pages/StandardsDetailsPage.tsx

```diff
@@ -28,7 +36,26 @@ export const standardDetailsPageContent: DetailsPageContentType = [
     id: 'About The Standard',
     title: 'About The Standard',
     element: (
-      <MarkdownSynapseFromColumnData columnName={'studyDescriptionLocation'} />
+      <DetailsPageContextConsumer columnName={'Name'}>
+        {({ value }) => (
+          <CardMetadataDisplay
+            sql={dataSql}
+            labels={[
+              'Name',
+              'Collections',
+              'Organizations',
+              'Data_Topic',
+              'Acronym',
+              'Is_Open',
+              'Registration',
+            ]}
+            columnAliases={dataColumnAliases}
+            searchParams={{
+              Name: value!,
+            }}
+          />
+        )}
```

Comment:
You actually already have the RowSet and Row in the DetailsPage context. In another comment I suggest splitting the component so you don't always have to issue a new table query. Here, you would use the component that doesn't create a new query

```suggestion
        {({ value, context }) => (
          <RowDataTable
            rowData={context.rowData}
            headers={context.rowSet.headers}
            labels={[
              'Name',
              'Collections',
              'Organizations',
              'Data_Topic',
              'Acronym',
              'Is_Open',
              'Registration',
            ]}
            columnAliases={dataColumnAliases}
          />
        )}
```

Reply:
Where exactly does rowData and rowSet come from? I see we can pass in a columnName to filter by but how do we know exactly what row to return?